### DISCUSSION TITLE
Jenkins Config

### DISCUSSION THREAD

**Discussion by AntaeusNar** [INTI-CMNB/kicad_ci_test#13] [Q&A] [answered]
Can you point me in right direction for utilizing this in a Jenkins CI/CD using a Jenkinsfile and the docker?

**Comment by set-soft** [comment]
Hi @AntaeusNar !
I never used Jenkins CI/CD. The name of the docker image is `setsoft/kicad_auto:latest`

**Comment by AntaeusNar** [comment, marked as answer]
Hey @set-soft  !

by adding
```
agent {
docker {label 'docker' image 'setsoft/kicad_auto:latest'}
} 
```
into my Jenkinsfile I was able to pull and access the docker! Now I just need to spend more time using it and converting your instruction for the GitHub actions into Jenkins CI/CD syntax.

Would you mind leaving this issue open as a place for me to update, and then that way the final method can be added to your readme?

**Comment by set-soft** [comment]
Hi @AntaeusNar! I'll leave it open.
